bitrise.yml

```diff
@@ -519,6 +389,7 @@ workflows:
     meta:
       bitrise.io:
         stack: osx-xcode-12.5.x
+        machine_type_id: g2.4core
     after_run:
     - RunSmokeXCUITests
   benchmark_test:
```

Comment:
This workflow can be removed to if you want to do it in this PR if not we will do it later in a follow up. Thanks!

Reply:
@isabelrios I removed this as well :)